docs-source/content/faq/external-clients.md

```diff
@@ -29,15 +29,15 @@ Here are the steps:
 
 - Review the [Security notes](#security-notes).
 
-##### Kubernetes NodePorts
+##### Kubernetes `NodePorts`
 
-The Kubernetes NodePorts approach for giving external WebLogic EJB or JMS clients access to a Kubernetes hosted WebLogic cluster involves configuring a network channel on the desired WebLogic cluster that accepts T3 protocol traffic, and deploying a Kubernetes NodePort that redirects external network traffic on the Kubernetes nodes to the network channel.
+The Kubernetes `NodePorts` approach for giving external WebLogic EJB or JMS clients access to a Kubernetes hosted WebLogic cluster involves configuring a network channel on the desired WebLogic cluster that accepts T3 protocol traffic, and deploying a Kubernetes `NodePort` that redirects external network traffic on the Kubernetes nodes to the network channel.
 
```

Comment:
I would add a note:  The NodePort approach is only available when worker nodes are accessible by the clients, for example when they have public IP addresses.  If private worker nodes are used and access to them is only possible through a load balancer or bastion, then the NodePort approach is not a valid option to provide access to external clients.